ehcache-core/src/main/java/org/ehcache/core/spi/store/tiering/AuthoritativeTier.java

```diff
@@ -75,7 +76,9 @@ public interface AuthoritativeTier<K, V> extends Store<K, V> {
    */
   void setInvalidationValve(InvalidationValve valve);
 
-  /**
+  Iterable<? extends Map.Entry<? extends K,? extends ValueHolder<V>>> bulkComputeIfAbsentAndFault(Iterable<? extends K> keyParam, Function<Iterable<? extends K>, Iterable<? extends Map.Entry<? extends K,? extends V>>> mappingFunction) throws StoreAccessException;
```

Comment:
I think we can javadoc this now that the signatures have stabilized.

Reply:
I have added method level javadoc